overlord/overlord.go

```diff
@@ -134,6 +135,9 @@ func New() (*Overlord, error) {
 
 	o.addManager(cmdstate.Manager(s))
 
+	o.stateEng.AddManager(NewUnknownTaskManager(s, o.knownTasks))
+	o.knownTasks = nil // not needed anymore
```

Comment:
It feels less surprising and less error prone to keep it around. Otherwise everybody needs to magically keep in mind the exact moment in which this will disappear before knowing whether they can trust this data or not, which is a known thing to yield very curious bugs.

In fact, instead of keeping it around, how about this: let's create the unknown task manager first as o.unknownMgr, and keep _that_ around. Then, inside addManager we can run:
```
o.unknownMgr.Ignore(mgr.KnownTaskKinds())
```
Which builds the list as needed instead of at once.

Reply:
Nice idea,thanks! Done.